### ISSUE TITLE
procstat memory fields missing in aggregation

### ISSUE THREAD

**Issue by adrianlzt** [influxdata/telegraf#4293]
### Relevant telegraf.conf:
```
[[outputs.file]]
  files = ["stdout"]

[agent]
  interval = "3s"
  flush_interval = "3s"

[[inputs.procstat]]
  pattern = "nginx: worker"
  fieldpass = ["pid","memory_rss"]

[[aggregators.minmax]]
  period = "3s"
```

### System info:
Telegraf v1.7.0 (git: release-1.7 f4d22dd4)

Arch Linux

### Steps to reproduce:

1. ``telegraf -config telegraf.conf``

### Expected behavior:
All fields from procstat being processed by the aggregator

### Actual behavior:
Only ``pid`` field gets computed by the aggregator.
```
procstat,host=archer,pattern=nginx:\ worker,process_name=nginx pid=29094i,memory_rss=1949696i 1529058174000000000                                                                                                  
procstat,host=archer,pattern=nginx:\ worker,process_name=nginx pid=29080i,memory_rss=1949696i 1529058174000000000                                                                                                  
procstat,host=archer,pattern=nginx:\ worker,process_name=nginx pid=29084i,memory_rss=1949696i 1529058174000000000                                                                                                  
procstat,host=archer,pattern=nginx:\ worker,process_name=nginx memory_rss=1949696i,pid=29094i 1529058177000000000                                                                                                  
procstat,host=archer,pattern=nginx:\ worker,process_name=nginx pid=29080i,memory_rss=1949696i 1529058177000000000                                                                                                  
procstat,host=archer,pattern=nginx:\ worker,process_name=nginx memory_rss=1949696i,pid=29084i 1529058177000000000                                                                                                  
procstat,host=archer,pattern=nginx:\ worker,process_name=nginx pid_min=29080,pid_max=29094 1529058177000000000
```

### Additional info:
With the procstat input, only the ``memory_*`` fields are not processed.

**Comment by adrianlzt** [comment]
[``convert``](https://github.com/influxdata/telegraf/blob/master/plugins/aggregators/minmax/minmax.go#L104) function fails if the number is not float64 or int64, but memory fields, comming from psutil lib, [are uint64](https://github.com/shirou/gopsutil/blob/master/process/process.go#L43)

**Comment by danielnelson** [comment]
Closed by #4294
